### PR TITLE
[doc] Release 0.6.1

### PR DESCRIPTION
**Pull request by ejgallego**
@palmskog do you mind having a look?

### CODE REVIEW

File: notes/announce-0.6.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+Subject: [ANNOUNCE] jsCoq and SerAPI releases
+
+Dear Coq users and developers,
+
+we are happy to announce the releases of jsCoq 0.9.3 and SerAPI 0.6.1
+for Coq 8.9.
+
+JsCoq and SerAPI are free software, please don't hesitate to contribute at:
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
JsCoq and SerAPI are free software, please don't hesitate to report issues and contribute at:
```

File: notes/announce-0.6.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+- https://github.com/ejgallego/jscoq
+- https://github.com/ejgallego/coq-serapi
+
+## JsCoq 0.9:
+
+JsCoq allows to interact with Coq developments as self-contained HTML,
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
JsCoq allows interacting with Coq developments as standard-compliant HTML documents using a browser.
```

File: notes/announce-0.6.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+standard-compliant web pages. The project aims to ease the development
+of interactive Coq documents and teaching material, to improve the
+accessibility of the platform, and to explore new user interaction
+possibilities.
+
+jsCoq 0.9 has been 3 years in the making, and it features a long list
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
jsCoq 0.9 has been three years in the making, and it features a long list
```

File: notes/announce-0.6.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+of improvements, most of them due to the incredible work of Shachar
+Itzhaky, who managed to stabilize the Web Worker version and provided
+countless usability and display improvements including
+"company-coq"-like and contextual information facilities.
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
"Company-Coq"-like contextual information display facilities.
```

File: notes/announce-0.6.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+While there are still some minor bugs, we feel that this is the first
+release that seems ready for wider testing and exposure; in
+particular, as Coq now runs in a separate browser thread, the overall
+experience is much smoother. Keep in mind that jsCoq's primary target
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
experience is much smoother than before. Keep in mind that jsCoq's primary target
```

File: notes/announce-0.6.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- https://x80.org/rhino-coq/examples/lf/
+- https://x80.org/rhino-coq/examples/plf/
+
+See more examples and information in the project's web page
+https://github.com/ejgallego/jscoq, the full list of changes is
+at https://github.com/ejgallego/jscoq/blob/v8.9%2Bworker/CHANGES.md
+
+## SerAPI 0.6.1
+
+Coq-SerAPI provides a S-expression based API for interaction with
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
Coq-SerAPI provides an S-expression based API suitable for machine interaction with
```

File: notes/announce-0.6.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+Coq. Capabilities include full round-trip serialization of Coq's AST
+and most internal structures, easy access to the document build and
+checking API, etc...
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
checking API, and facilities for querying Coq's environment and proof state.
```

File: notes/announce-0.6.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+The 0.6.1 release brings many improvements and features in the
+serialization front. Thanks to the awesome efforts of Karl Palmskog,
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
serialization front. Thanks to the efforts of Karl Palmskog and Ahmet Celik,
```

File: notes/announce-0.6.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+SerAPI is now able to serialize/deserialize large Coq developments
+reliably ; we have also improved the `sercomp` compiler tool, which is
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
reliably, and we have improved the `sercomp` command-line tool for batch (de)serialization of Coq files.
```

File: notes/announce-0.6.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+now a more mature tool.
+
+See more examples and information in the project's web page
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
See more information and examples on the project's website
```

File: notes/announce-0.6.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+https://github.com/ejgallego/coq-serapi, the full list of changes is
+at https://github.com/ejgallego/coq-serapi/blob/v8.9/CHANGES.md
+
+Best regards,
+Emilio & all the people involved in these projects
```
Comment (palmskog) [inline notes/announce-0.6.md]:
```suggestion
Emilio & all contributors to these projects
```

File: notes/announce-0.6.md (hunk 2 above)
Comment (palmskog) [reply notes/announce-0.6.md]: Maybe it should be "Coq documents", I think we are trying to push that terminology?

File: notes/announce-0.6.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+interaction with Coq. Capabilities include full round-trip
+serialization of Coq's AST and most internal structures, easy access
+to the document build and checking API, and facilities for querying
+Coq's environment and proof state.
+
+The 0.6.1 release brings many improvements and features in the
+serialization front. Thanks to the awesome efforts of Karl Palmskog
+and Ahmet Celik, SerAPI is now able to serialize/deserialize large Coq
+developments reliably, and we have improved the `sercomp` command-line
+tool for batch (de)serialization of Coq files.  now a more mature
```
Comment (palmskog) [inline notes/announce-0.6.md]: "now a more mature" and dangling "tool." is wrong here, I couldn't figure out how to do multi-line suggestions...

Comment (ejgallego) [reply notes/announce-0.6.md]: Ups, thanks a lot!

**Pull request merged**